src/VirtoCommerce.CatalogModule.Core/Model/Search/DynamicAssociationSearchCriteria.cs

```diff
@@ -0,0 +1,9 @@
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.CatalogModule.Core.Model.Search
+{
+    public class DynamicAssociationSearchCriteria : SearchCriteriaBase
+    {
```

Comment:
Please add StoreIds and Groups to the search criteria, and use them in the search service query

Reply:
done

Reply:
Please add them as string arrays

Reply:
Same for the groups